bemani/backend/museca/museca1plus.py

```diff
@@ -180,12 +181,59 @@ def enable_event(eid: int) -> None:
         if self.omnimix:
             for evtid in museca_plus_events:
                 enable_event(evtid)
+        
+        #TODO: Add the campaign events to the webui so players can enable them in their profile.
+        # List of known event IDs:
+            # 56,   # Generator grafica icon <print 1 in musicdb>
+            # 83,   # Paseli Light Start
+            # 86,   # Generator grafica icon <print 2 in musicdb>
+            # 98,   # Caption 2 notice (grs_grafica_caption_2.png)
+            # 100,  # DJ YOSHITAKA EXHIBITION 2016
+            # 103,  # HATSUNE MIKU EXHIBITION 2016 - PART 1
+            # 104,  # HATSUNE MIKU EXHIBITION 2016 - PART 2
+            # 105,  # Makes the "Number of Layers" option visible in game settings
+            # 106,  # HATSUNE MIKU EXHIBITION 2016 - PART 3
+            # 117,  # NEW GENERATION METEOR DIFFUSE FESTA 2016 / RYUSEI FESTA TRIGGER
+            # 129,  # COCONATSU EXHIBITION 2016
+            # 130,  # Curator Rank
+            # 97,   # Agetta Moratta (vmlink_phase 1 in musicdb)
+            # 114,  # Agetta Moratta (vmlink_phase 2 in musicdb)
+            # 140,  # Agetta Moratta (vmlink_phase 3 in musicdb)
+            # 141,  # Coconatsu & Mukipara grafica effects
+            # 143,  # Matching
+            # 144,  # BEMANI ARCHAEOLOGICAL EXHIBITION
+            # 163,  # TUTORIAL SNOW
+            # 169,  # SHIORI FUJISAKI EXHIBITION 2017 - PART 1
+            # 174,  # SHIORI FUJISAKI EXHIBITION 2017 - PART 2
+            # 182,  # Mute illil's voice?
+            # 192,  # GREAT REPRINT FESTIVAL: MIKU + DJ YOSHITAKA
+            # 194,  # Continue
+            # 195,  # Fictional Curator (foot pedal options)
+            # 211,  #News 1
+            # 212,  #News 2
+            # 213,  #News 3
+            # 214,  #News 4
+            # 217,  #News 5
+            # 218,  #News 6
+            # 219,  #News 7
+            # 220,  #News 8
+            # 221,  # GRAFICA PRESENTATION CAMPAIGN “THE PRIMITIVE LIFE EXHIBITION”
+            # 222,  # GRAFICA PRESENTATION CAMPAIGN "NOISE"
+            # 223,  # GRAFICA PRESENTATION CAMPAIGN "PATISSERIE ROUGE"
+            # 224,  # GRAFICA PRESENTATION CAMPAIGN "GUNSLINGER"
+            # 145,  # MUKIPARA UNLOCKS
+            # 146,  # MUKIPARA UNLOCKS
+            # 147,  # MUKIPARA UNLOCKS
+            # 148,  # MUKIPARA UNLOCKS
+            # 149,  # MUKIPARA UNLOCKS
+        
+        
         # Makes special missions available on grafica that have them.
         extend = Node.void('extend')
         game.add_child(extend)
         info = Node.void('info')
         extend.add_child(info)
-        info.add_child(Node.u32('extend_id', 12))
+        info.add_child(Node.u32('extend_id', 1))
```

Comment:
Is the ID just what the game refers to this node with, as defined by the server? Or is it tied to a specific thing in-game?

Reply:
The ID is just what the game refers to the node with, it can be anything. No reason for it to start at 12, that was an oversight on my part when I gave subject the code originally.